packages/core/test/unit/query-generator/select-query.test.ts

```diff
@@ -508,14 +508,9 @@ Only named replacements (:name) are allowed in literal() because we cannot guara
   });
 
   describe('minifyAliases', () => {
-    const minifyAliasesSequelize = createSequelizeInstance({
```

Comment:
Could we still have the unit test for the global option? So we keep testing both in the unit tests.

And we should also test the local option in the integration tests. A duplicate of the current test cases should be enough I think. For the next PR we should add see if we can add `minifyAliases` to `DialectSupports` and move the tests to use that so when we add support for new dialects the tests are also run automatically

Reply:
That should go in queryInterface tests in integration testing
Can't have the global option here, queryGenerator doesn't use it

Reply:
Ah yes, that's true. Then just integration tests for queryInterface would be fine for now. We can see about adding it to queryInterface unit tests later as part of improving test suite performance